src/Mod/TechDraw/App/Cosmetic.cpp

```diff
@@ -280,20 +272,14 @@ boost::uuids::uuid CosmeticVertex::getTag() const
 
 std::string CosmeticVertex::getTagAsString(void) const
 {
-    std::string tmp = boost::uuids::to_string(getTag());
-    return tmp;
+    return boost::uuids::to_string(getTag());
 }
 
 void CosmeticVertex::createNewTag()
 {
     // Initialize a random number generator, to avoid Valgrind false positives.
     static boost::mt19937 ran;
-    static bool seeded = false;
-
-    if (!seeded) {
-        ran.seed(static_cast<unsigned int>(std::time(nullptr)));
-        seeded = true;
-    }
+    ran.seed(static_cast<unsigned int>(std::time(nullptr)));
```

Comment:
@WandererFan Can you please make sure that this doesn't do any harm?

Reply:
This doesn't do the same thing as the replaced code.  seeded is static so ran.seed only gets called the first time through.   The new code resets the random number generator every time.

This same code is in Part and PartDesign, so I would leave it as is.